.ci/containers/terraform-tester/test_terraform.sh

```diff
@@ -66,22 +66,22 @@ curl \
 
 set +e
 
-make
```

Comment:
I think we need to keep this line so that we catch code compilation issues as part of linting.

Reply:
added --- oops didn't mean to remove this ! good catch !!